HIRS_AttestationCA/src/main/java/hirs/attestationca/service/SupplyChainValidationServiceImpl.java

```diff
@@ -214,6 +215,19 @@ public SupplyChainValidationSummary validateSupplyChain(final EndorsementCredent
             }
         }
 
+        if (policy.isFirmwareValidationEnabled()) {
+            // may need to associated with device to pull the correct info
+            // compare tpm quote with what is pulled from RIM associated file
+            IssuedAttestationCertificate attCert = IssuedAttestationCertificate
+                .select(this.certificateManager)
+                .byDeviceId(device.getId())
+                .getCertificate();
+
+            if (attCert != null) {
```

Comment:
This is an intentionally left in debug statement.  I cleaned up a lot of my log statements but left this one in because either checkstyles or find bugs doesn't like empty if statements.  And this will be where the start of the validation code will behind for the second part. Error is used because otherwise I would have to change the logging properties file which would cause me to restart tomcat.